_templates/sdk/new/lib-index.ts.t

```diff
@@ -5,15 +5,14 @@ to: packages/sdk-<%= name %>/lib/index.ts
 import itlySdk, {
   ItlyOptions,
```

Comment:
Can you please remove the `Itly` prefix from these imports. It's a legacy issue but would be nice to clean up.

Reply:
Done